tests/test_mi.py

```diff
@@ -245,18 +265,40 @@ def test_machine_learning_error_should_be_normalized(self):
 })
 class ServiceFabricTestCase(ClientTestCase):
 
-    def _test_happy_path(self, app):
+    def _test_happy_path(self, app, *, claims_challenge=None) -> callable:
         expires_in = 1234
         with patch.object(app._http_client, "get", return_value=MinimalResponse(
             status_code=200,
             text='{"access_token": "AT", "expires_on": %s, "resource": "R", "token_type": "Bearer"}' % (
                 int(time.time()) + expires_in),
         )) as mocked_method:
             super(ServiceFabricTestCase, self)._test_happy_path(
-                app, mocked_method, expires_in)
+                app, mocked_method, expires_in, claims_challenge=claims_challenge)
+            return mocked_method
 
-    def test_happy_path(self):
-        self._test_happy_path(self.app)
+    def test_happy_path_with_client_capabilities_should_relay_capabilities(self):
```

Comment:
can you also pass an empty list client_capabilities=[], and confirm that xms_cc is omitted? did not see one.

Reply:
Fair point. Added it into the next test case.